### ISSUE TITLE
Create a wiki page describing the order of implementing features

### ISSUE THREAD

**Issue by shhmon** [teknik-eksjo/chronos#119]
At least for me, it's getting kind of hard to follow what can be and what cannot be done right now. I'd like us to use the time we spend together in a classroom to create some sort of planning or description of what the individual features are dependent on - using the wiki. I know this is kind of the purpose of milestones but with soon 40 issues, personally they aren't helping me at all. I'd like to get a quick overview of what is needed to start working on this or that feature. After all, the wiki is for 

>  planning and arrangement of the project.

Is this a good idea? Thoughts @teknik-eksjo/te13 ?

**Comment by Greenheart** [comment]
I agree. 

@teknik-eksjo/te13 What do you say about **going through all current issues** and **make sure they're in the right milestone** during the lesson tomorrow?

**A tip:** As long as the milestones contain the right issues for the specific stage of the project, it's possible to filter out issues for the current milestone by using [this link](https://github.com/teknik-eksjo/chronos/milestones) or simply clicking **Issues** --> **Milestones** and then selecting the one you want to filter out.

Another suggestion for what we could do to make the prioritization of issues easier is to **use labels**. For example, adding `In progress` for the ones currently being worked on or `Help wanted` on those where help is needed would make it easier to see what issues are currently WIP and should be prioritized.
